src/features/onboarding/newPassword/NewPassword.styles.ts

```diff
@@ -1,10 +1,8 @@
 import styled from 'styled-components'
```

Comment:
Please convert this component to TailwindCSS and remove the `styled-components` OR remove it if we no longer used

Reply:
fixed